### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by skcihmit** [streamlink/streamlink#3466] [General]
Congrats on 2.0. I saw that I can now monetarily contribute to the project. Does the group have some guiding principles on how the contributions are divided by the team?

**Comment by gravyboat** [comment]
Right now we don't do any sort of contribution splitting. The [Open Collective](https://opencollective.com/streamlink) is used for project related expenses when they come up. If you want to contribute to a specific individual you can either sponsor them directly via GitHub if they have it set up or contact them directly. There's also the [Donate](https://streamlink.github.io/donate.html) page available from the docs, but I'm not sure how up to date that is for everyone.

There have been 2 expenses thus far that were paid out for from donated funds:

1. Bintray costs were putting undo pressure on Beardypig so they were reimbursed from our Bountysource (this was before the Open Collective was set up and before we [transferred away from them](https://github.com/streamlink/streamlink/issues/1388).
2. We paid a community member for logo design which you can [see as approved in the expense report here](https://opencollective.com/streamlink/expenses).

Outside of that there are no specific principles set up.
